chromium_src/components/privacy_sandbox/DEPS

```diff
@@ -2,5 +2,6 @@ specific_include_rules = {
   "privacy_sandbox_settings_unittest.cc": [
     "+brave/components/privacy_sandbox",
     "+chrome/test/base/testing_profile.h",
+    "+content/public/browser/content_browser_client.h"
```

Comment:
we really need to fix this test, not add more exceptions, but what needed this header in the first place? Also shouldn't we be overriding PrivacySandboxSettingsDelegate? There's technically nothing wrong with adding ContentBrowserClient here, but we shouldn't have things like testing_profile here

Reply:
cc @pes10k

Reply:
also it seems like that really testing that this is truly disabled would be better done by doing something similar to `PrivacySandboxSettingsBrowserTest`, not a unit test

Reply:
this would be for follow-up